edx_proctoring/management/commands/set_is_attempt_active.py

```diff
@@ -0,0 +1,102 @@
+"""
+Django management command to update the is_attempt_active field on
+ProctoredExamSoftwareSecureReview and ProctoredExamSoftwareSecureReviewHistory models
+"""
+import logging
+import time
+
+from django.core.management.base import BaseCommand
+
+from edx_proctoring.models import (
+    ProctoredExamSoftwareSecureReview,
+    ProctoredExamSoftwareSecureReviewHistory,
+    ProctoredExamStudentAttempt
+)
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django Management command to update is_attempt_active field on review models
+    """
+    update_field_count = 0
+    update_attempt_codes = []
+    distinct_attempt_codes = set()
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch_size',
+            action='store',
+            dest='batch_size',
+            type=int,
+            default=300,
+            help='Maximum number of attempt_codes to process. '
+                 'This helps avoid locking the database while updating large amount of data.')
```

Comment:
#nit: I would move the closing bracket to the new line which is much clear for reading and adding a trailing comma to ensure diff for the future updates won't have useless strings.